Library/Formula/git.rb

```diff
@@ -1,7 +1,7 @@
 class Git < Formula
   homepage "http://git-scm.com"
```

Comment:
Na. Opportunistic changes are fine in PRs, especially plaintext => SSL/TLS link upgrades.